frontend/crd/src/domain/Task.ts

```diff
@@ -1,20 +1,23 @@
 export interface TaskJSON {
-    description: string;
-    needsArtifact: Boolean;
-    id: string;
-    isRequired: Boolean;
+  description: string;
+  needsArtifact: boolean;
+  id: string;
+  isRequired: boolean;
```

Comment:
aren't all tasks required? when would a task not be required?

Reply:
the task can be optional for completing the milestone